types/office-js/index.d.ts

```diff
@@ -9330,7 +9330,7 @@ declare namespace Office {
          * {@link https://learn.microsoft.com/javascript/api/outlook/office.spamreportingeventcompletedoptions#outlook-office-spamreportingeventcompletedoptions-postprocessingaction-member |
         * Office.SpamReportingEventCompletedOptions.postProcessingAction}.
         */
-       enum MoveSpamItemTo {
+        enum MoveSpamItemTo {
```

Comment:
no need to change this spacing

```suggestion
       enum MoveSpamItemTo {
```